### PR TITLE
Enable phpstan analyses on templates files

### PR DESCRIPTION
**Pull request by CauanCabral**
Fix all detected issues

### CODE REVIEW

**Comment by ADmad** [comment]
Doing static analysis on template files is an overkill.

There are no actual issues in the template code and most of the changes are quite unnecessary.

**Comment by CauanCabral** [comment]
I almost agree, but I think at least [this one](https://github.com/cakephp/app/pull/831/files#diff-1f1107533aee24daa1e811240d4a9f9a702040350a602340ec320ed3eede9046L151) is a bug.

`Exception::getMessage()` should return string, so I think `$errorMsg` is a typo for `$attributes`.

File: phpstan.neon [hunk 1]
```diff
@@ -2,7 +2,10 @@ parameters:
     level: 7
     checkMissingIterableValueType: false
     treatPhpDocTypesAsCertain: false
+    bootstrapFiles:
+        - config/paths.php
     paths:
         - src
+        - templates
```
Comment (markstory) [inline phpstan.neon:9]: Needing to set `@var` in all the templates is going to be tedious, can we skip this change?

**Comment by ADmad** [comment]
> Exception::getMessage() should return string, so I think $errorMsg is a typo for $attributes.

Then we can fix just that particular line.

In a full app running static analysis on templates will most likely return a lot of false positives, due to lack of context. Having to add `@var` tags for all view variables used in a template would be highly annoying.

**Comment by dereuromark** [comment]
> In a full app running static analysis on templates will most likely return a lot of false positives, due to lack of context. Having to add @var tags for all view variables used in a template would be highly annoying.

IdeHelper helps doing that, but I agree that this should rather stay opt-in for now.

**Comment by CauanCabral** [comment]
Fixed with #832

**Pull request closed without merging**